server/database/queries/eventPost.js

```diff
@@ -0,0 +1,13 @@
+const dbconniction = require('../db_connection')
+
+const postEvent = (data) => {
+  const sql = {
+    text: 'INSERT INTO events (event_title ,event_descriprion ,event_note,event_date,user_id,room_id) VALUES ($1,$2,$3,$4,$5,$6)',
```

Comment:
`event_descriprion` should probably be `event_description`, also, pls rename `dbconniction` to `dbConnection`